tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -753,3 +753,12 @@ vxlan/test_vxlan_ecmp.py:
     reason: "VxLAN ECMP test is not yet supported on multi-ASIC platform. Also this test can only run on 4600c and 8102."
     conditions:
       - "(is_multi_asic==True) or (platform not in ['x86_64-mlnx_msn4600c-r0', 'x86_64-8102_64h_o-r0'])"
+
+#######################################
+#####         test_ser            #####
+#######################################
+platform_tests/broadcom/test_ser.py:
+  skip:
+    reason: "test_ser is only supported for Broadcom"
+    conditions:
+      - "asic_type not in ['broadcom']"
```

Comment:
Can you move this skip condition into another file `tests/common/plugins/conditional_mark/tests_mark_conditions_platform_tests.yaml` and ensure alphabetic order?

Reply:
addressed, please re-review